benchmarks/workloads.py

```diff
@@ -133,3 +137,56 @@ def large_dense_attr_hex(cls, x: int = 1024, y: int = 1024) -> str:
             )
         ]
         return WorkloadBuilder.wrap_module(ops)
+
+    @classmethod
+    def large_constant_tensor(
+        cls,
+        tensor_shape: tuple[int, ...],
+        num_add_ops: int = 10,
+    ) -> ModuleOp:
+        """Create a module with a function that adds multiple large constant tensors.
+
+        Args:
+            tensor_shape: The shape of the constant tensors to be created.
+            num_add_ops: The number of addition operations to perform with the constant tensors.
+
+        Returns:
+            The created module containing the function with addition operations.
+        """
+        assert num_add_ops >= 0
+        random.seed(RANDOM_SEED)
+        tensor_type = TensorType(shape=tensor_shape, element_type=Float32Type())
+        function_type = FunctionType.from_lists(
+            inputs=[tensor_type], outputs=[tensor_type]
+        )
+
+        func_op = FuncOp(name="main", function_type=function_type)
```

Comment:
It feels like the `func_op` isn't really necessary to what we're trying to show here, but I guess it's fine to keep it.